orchestra/utils/task_properties.py

```diff
@@ -70,22 +70,3 @@ def all_workers(task):
             A list of all workers involved with `task`.
     """
     return [assignment.worker for assignment in assignment_history(task).all()]
-
```

Comment:
we should move these functions over the mixins as well, just add a TODO and we'll do it in another PR